mobile/android/scripts/build_target.sh

```diff
@@ -80,13 +80,19 @@ export LDFLAGS="${LDFLAGS} -L${SYSROOT}/usr/lib${LIB_SUFFIX} -L${ANDROID_TOOLCHA
     test -f Makefile && make clean
     echo "Configure with --host=${ARCH} and toolchain ${ANDROID_TOOLCHAIN}"
     test -x ${ROOTDIR}/../../configure || (cd ${ROOTDIR}/../.. && autoreconf -i)
-    ${ROOTDIR}/../../configure --host=${ARCH} --with-sysroot=${SYSROOT} \
+    ${ROOTDIR}/../../configure -q --host=${ARCH} --with-sysroot=${SYSROOT} \
       --with-libevent=builtin --with-yaml-cpp=builtin \
       --with-boost=builtin --disable-shared --libdir=/ \
       --includedir=/include --with-libmaxminddb=builtin \
       --with-jansson=builtin
     make V=0
     echo "Installing library in ${BASEDIR}/build/${ANDROID_TOOLCHAIN}"
+    make install-strip DESTDIR=${ROOTDIR}/jni/${DESTDIR_NAME}
+    rm -rf ${ROOTDIR}/jni/${DESTDIR_NAME}/include
     make install-data-am DESTDIR=${ROOTDIR}/jni/${DESTDIR_NAME}
-    make install-exec DESTDIR=${ROOTDIR}/jni/${DESTDIR_NAME}
+    rm -rf ${ROOTDIR}/jni/${DESTDIR_NAME}/pkgconfig
+    rm -rf ${ROOTDIR}/jni/${DESTDIR_NAME}/usr
+    rm -rf ${ROOTDIR}/jni/${DESTDIR_NAME}/*.la
+    rm -rf ${ROOTDIR}/jni/${DESTDIR_NAME}/libevent_core.a
+    rm -rf ${ROOTDIR}/jni/${DESTDIR_NAME}/libevent_extra.a
```

Comment:
I forgot to explain this. Here the idea is:

1) do `install-strip` because otherwise I need to figure out the proper strip command for each architecture and at the first glance this was not straightforward (I noticed multiple strip commands for some architectures)

2) since step 1 installs *all* headers, including the ones installed by dependencies, next I remove all headers and run again the specific subcommand need to install headers only

3) then I remove extra stuff that is installed (pkgconfig, usr), libtool files (not needed), and two libevent libraries that we don't use

Sorry for not specifying this in the commit message. @DavideAllavena do you want me to document this in the source code?